classes/shaman.lua

```diff
@@ -16,6 +16,7 @@ local RollingThunderHandler = {
     earthShockSpells = {},
     -- Variables
     initialized = false,
+    glowtimer,

```

Comment:
This does not really do what you want. It adds the value of `glowtimer` (which is `nil`) to the indexed list of `RollingThunderHandler`.

You can try this out in the game chat:
```lua
/dump { a="a", b, c="c", 1, 2 }
```
You will see that `1` and `2` are stored at index 2 and 3. That's because `b` was inserted with a `nil` value at the first index. The above line is (almost) equivalent to:
```lua
/dump { ["a"]="a", [1]=b, ["c"]="c", [2]=1, [3]=2 }
```
Which is equivalent to the following, because `b` is not defined:
```lua
/dump { ["a"]="a", [1]=nil, ["c"]="c", [2]=1, [3]=2 }
```

To tell Lua that `glowtimer` is a variable, it should be initialized something, even if this _something_ is _nothing_ (i.e. `nil`):
```lua
glowtimer = nil,
```

Reply:
Done in 773860f39b174ce23bab992e81d957455314d387.